### DISCUSSION TITLE
how to upload photo?

### DISCUSSION THREAD

**Discussion by momomobinx** [Radiokot/photoprism-android-client#138] [Q&A] [answered]
how to upload photo?

**Comment by ippocratis** [comment, marked as answer]
> how to upload photo?

The app it self does not offer upload functionality.

You have to use the webapp to upload photos

**Reply by Radiokot** [reply]
The ability to quickly import photo(s) by sending them to the gallery is in the backlog and will be implemented eventually.

**Reply by Radiokot** [reply]
Import through sharing is added in 1.28.0. Open the app once after the installation so the component gets enabled
https://github.com/Radiokot/photoprism-android-client/releases/tag/1.28.0(42)
